web/libs/editor/src/components/Comments/Comment/CommentItem.scss

```diff
@@ -0,0 +1,142 @@
+@keyframes pulse {
+  0% {
+    transform: scale(1);
+  }
+
+  50% {
+    transform: scale(0.5);
+  }
+}
+
+.comment-item {
+  --comment-item-padding: 8px;
+  --comment-item-spacing: 6px;
+  --comment-item-content-font-size: 14px;
+  --comment-item-content-line-height: 20px;
+  --comment-item-content-letter-spacing: 0.25px;
+  --comment-item-avatar-size: 24px;
+  --comment-item-name-color: var(--sand_900);
+  --comment-item-content-color: var(--sand_900);
+  --comment-item-content-color-resolved: var(--sand_500);
+  --comment-item-icon-color-resolved: var(--success_color);
+  --comment-item-region-opacity: 1;
```

Comment:
❤️ 
this is great!

Reply:
Oh, it mostly was already there before 😊 https://github.com/HumanSignal/label-studio/blob/develop/web/libs/editor/src/components/Comments/CommentItem.scss#L12-L21